### PR TITLE
refactor: merge two transaction rpc interface

### PR DESCRIPTION
**Pull request by driftluo**
1. Modify the return value of the `get_transaction` interface
2. Modify the return type of the `get_transaction`

### CODE REVIEW

**Comment by yangby-cryptape** [comment]
Could u update the [documents](https://github.com/nervosnetwork/ckb/blob/e1283e4fbaf65afbb98fa3ff103598f3d190f96b/rpc/README.md), please?

**Comment by driftluo** [comment]
> Could u update the [documents](https://github.com/nervosnetwork/ckb/blob/e1283e4fbaf65afbb98fa3ff103598f3d190f96b/rpc/README.md), please?

done

File: util/jsonrpc-types/src/blockchain.rs [hunk 1]
```diff
@@ -232,6 +232,54 @@ impl TryFrom<Transaction> for CoreTransaction {
...
+            tx_status: TxStatus::Committed(hash),
+            transaction: (&tx).into(),
+        }
+    }
+}
+
+/// Can see the serialization results on the links: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=475702fa87bc741e5d482fca52ba139f
+#[derive(Clone, Serialize, Deserialize, PartialEq, Eq, Hash, Debug)]
+#[serde(tag = "status", content = "block_hash")]
+pub enum TxStatus {
```
Comment (yangby-cryptape) [inline util/jsonrpc-types/src/blockchain.rs]: Both are ok, but I think `#[serde(rename_all = "lowercase")]` is better.

**Comment by yangby-cryptape** [comment]
cc @ashchan @classicalliu

**Pull request merged**